src/mscorlib/shared/Interop/Windows/Kernel32/Interop.GetSystemTimeAsFileTime.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        [DllImport(Libraries.Kernel32)]
+        private extern static unsafe void GetSystemTimeAsFileTime(long* lpSystemTimeAsFileTime);
+
+        [DllImport(Libraries.Kernel32)]
+        private extern static unsafe void GetSystemTimePreciseAsFileTime(long* lpSystemTimeAsFileTime);
+
+        private unsafe delegate void GetSystemTimeAsFileTimeFn(long* lpSystemTimeAsFileTime);
+
+        private static Lazy<GetSystemTimeAsFileTimeFn> _getSystemTimeAsFileTime = new Lazy<GetSystemTimeAsFileTimeFn>(GetGetSystemTimeAsFileTimeFn);
```

Comment:
This method is performance sensitive. Going through delegate like this will likely make it several times slower. I do no think it is acceptable perf regression.